feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.ts

```diff
@@ -137,4 +144,41 @@ export class ConfiguratorAttributeHeaderComponent
     const images = this.attribute.images;
     return images && this.hasImage ? images[0] : undefined;
   }
+
+  navigateToGroup(): void {
+    this.configuratorCommonsService
+      .getConfiguration(this.owner)
+      .pipe(take(1))
+      .subscribe((configuration) => {
+        if (this.groupType === Configurator.GroupType.CONFLICT_GROUP) {
+          const groupId = this.attribute?.groupId;
+          if (groupId) {
+            this.configuratorGroupsService.navigateToGroup(
+              configuration,
+              groupId
+            );
+            this.focusAttribute(this.attribute.name);
+          }
+        }
+      });
+  }
+
+  protected focusAttribute(name: string): void {
```

Comment:
Please state that this is about the focus handling in the group that we are navigating to, so it does not affect the current view. Will make it easier to understand why we call the 'loading' method twice

Reply:
Done!